docs/Ecosystem.md

```diff
@@ -15,6 +15,7 @@ title: "Ecosystem"
 - [dreinke/aor-color-input](https://github.com/dreinke/aor-color-input): a color input using [React Color](http://casesandberg.github.io/react-color/), a collection of color pickers.
 - [LoicMahieu/aor-tinymce-input](https://github.com/LoicMahieu/aor-tinymce-input): a TinyMCE component, useful for editing HTML
 - [marmelab/aor-dependent-input](https://github.com/marmelab/aor-dependent-input): conditionnaly display inputs depending on other inputs values
+- [mhdsyrwan/aor-embedded-array](https://github.com/MhdSyrwan/aor-embedded-array): Field and Input for representing embedded arrays.
```

Comment:
s/for representing/for/

Reply:
like this? "Field and Input for embedded arrays."

Reply:
Done